presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -81,13 +89,35 @@ protected ResultSet getTables(Connection connection, String schemaName, String t
                 new String[] {"TABLE", "VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE"});
     }
 
+    public static ReadMapping jsonReadMapping()
```

Comment:
- `private`?
- move the method below `toPrestoType`

Reply:
will change to private and move it to right place